scripts/wallet.js

```diff
@@ -714,8 +733,12 @@ export class Wallet {
         getEventEmitter().emit('new-tx');
     });
 
-    async #transparentSync() {
-        if (!this.isLoaded() || this.#isSynced) return;
+    /**
+     * Synchronise UTXOs via xpub/address from the current explorer.
+     * @param {boolean} [force=false] - Force transparent sync, regardless of state
+     */
+    async #transparentSync(force = false) {
+        if ((!this.isLoaded() || this.#isSynced) && !force) return;
```

Comment:
Move the loadedCheck and synced in the sync function, so we can remove the force param.
```suggestion
    async #transparentSync() {
```
It's only going to be called by sync anyways

Reply:
I think we should change `checkShieldSaplingRoot` to call `sync`, right now it doesn't resync transparent if sapling root changes when MPW is open